docs/source/tutorials/preserve_ram.rst

```diff
@@ -0,0 +1,18 @@
+How to preserve RAM
+===================
+
+The amount of memory consumed can be measured in debug mode with the following lines of code.
+
+.. code-block:: python
+    
+    device.setLogLevel(dai.LogLevel.DEBUG)
+    device.setLogOutputLevel(dai.LogLevel.DEBUG)
+
+There might be some memory issues when creating a larger number (around 10) of image manip, config and manip output nodes.
```

Comment:
Rewrite:
> of ImageManip, XLinkIn and other nodes

Reply:
and since it's depthai-python, use refs (``:ref:`ImageManip` ``) to link to nodes.